components/button/button-subtle.js

```diff
@@ -59,19 +65,41 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				}
 
 				button {
+					--d2l-button-subtle-padding-left: 0.6rem;
+					--d2l-button-subtle-padding-right: 0.6rem;
```

Comment:
Suggestion: call these `--d2l-button-subtle-padding-inline-start` and `-inline-end`. This will align the terminology with the upcoming [CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) that we'll be switching to as soon as the browser support is there. It will let us remove the RTL-specific CSS.